content/en/docs/tasks/job/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
 title: "Run Jobs"
```

Comment:
So in Hugo you put in _index.md the title of the menu (i.e. "Run Jobs") and then you move all the pages that that are part of this menu into the job folder.  And then on each page you use an increasing weight value to establish the order of the pages in the menu.  This is very cool!  Thanks @steveperry-53 I learned a lot from this PR!

Reply:
This looks great and preserves the order of pages from the 1.9 version of the docs. Great job!
/lgtm
/approve